sherpa/io.py

```diff
@@ -185,8 +185,8 @@ def get_ascii_data(filename, ncols=1, colkeys=None, sep=' ', dstype=Data1D,
 
     Data lines are separated into columns - splitting by the
     ``sep`` comment - and then converted to NumPy arrays.
-    If the ``require_floats`` argument is ``True`` then the
-    column will be converted to the `sherpa.utils.SherpaFloat`
+    If the ``require_floats`` argument is `True` then the
+    column will be converted to a floating point number
```

Comment:
```suggestion
    column will be converted to a floating-point number
```

really-important change